### ISSUE TITLE
Check if the current design still holds up

### ISSUE THREAD

**Issue by menor** [resmio/widget#85]
It's like 18 months old already, so maybe @fabiennepimenta can take a look and update it to align it to our current style guide.

**Comment by fabiennepimenta** [comment]
yes, that was the final version. We can take a look on what is doable or not. And after that styles.

![artboard 21](https://cloud.githubusercontent.com/assets/3285116/18631173/b735a5d2-7e70-11e6-9560-8393e3c59d7e.png)
